### ISSUE TITLE
When applying overlays, if a rating is not found, it will place the text <<critic_rating#>> (or similar) on the image

### ISSUE THREAD

**Issue by nmorgowicz** [Kometa-Team/Kometa#1011]
### Version Number

1.17.3

### What branch are you on?

master

### Describe the Bug

I had overlays set up since before the 1.17.x line, and everything was working great until i upgraded from 1.17.2 to 1.17.3.

The way my overlay is supposed to work is to have <<audience_rating#>> on the bottomleft and <<critic_rating#>> on the bottomright, and on the cases where some tv/movies didn't have a rating, it would leave that part out and not show a number or my IMDB/RT colour-coded image.

However, when moving up to 1.17.3, if it does not find the rating, it will throw a warning when it tries to apply the overlay, and overlay the text <<critic_rating#>> or <<audience_rating#>>, whichever didn't have a number.

Here's a warning:
|====================================================================================================|
|                             Applying Overlays for the TV-Anime Library                             |
|====================================================================================================|
|                                                                                                    |
| Overlay Warning: No <<critic_rating#>> found                                                       |
| 7SEEDS                                                       | Overlays Applied: IMDB_G, audience_rating, critic_rating, Dolby-Digital, Dual-Audio, 1080P, Netflix |
![image](https://user-images.githubusercontent.com/7116802/186578386-bbcfce0c-3d32-4686-866a-849ffc204356.png)

My overlay ratings.yml is below.

### Relevant Collection/Overlay/Playlist Definition

```yml
overlays:
  IMDB_R:
    overlay:
      name: IMDB_R
    plex_search:
      all:
        audience_rating.lt: 7.0
  IMDB_Y:
    overlay:
      name: IMDB_Y
    plex_search:
      all:
        audience_rating.gte: 7.0
        audience_rating.lt: 8.0
  IMDB_G:
    overlay:
      name: IMDB_G
    plex_search:
      all:
        audience_rating.gte: 8.0
  RT_R:
    overlay:
      name: RT_R
    plex_search:
      all:
        critic_rating.lt: 6.0
  RT_Y:
    overlay:
      name: RT_Y
    plex_search:
      all:
        critic_rating.gte: 6.0
        critic_rating.lt: 8.0
  RT_G:
    overlay:
      name: RT_G
    plex_search:
      all:
        critic_rating.gte: 8.0
  audience_rating:
    overlay:
      name: text(audience_rating#)
      horizontal_offset: 112
      vertical_offset: 1390
      font: fonts/Inter-Medium.ttf
      font_size: 60
    plex_all: true
  critic_rating:
    overlay:
      name: text(critic_rating#)
      horizontal_offset: 804
      vertical_offset: 1390
      font: fonts/Inter-Medium.ttf
      font_size: 60
    plex_all: true
```

### Logs

https://gist.github.com/nickveldrin/20607a7a6419b20d50dc6d1f5faef57f

**Comment by YozoraXCII** [comment]
This is a known issue in 1.17.3 and we are currently awaiting a fix from Sohjiro.

**Comment by nmorgowicz** [comment]
> 

Oh! I had tried to look to see if someone reported it or if it was noted somewhere, but didn't see anything.

Thanks for the update.

**Comment by meisnate12** [comment]
should be fixed in 1.18.0
